tests/trinity/core/p2p-proto/test_state_sync.py

```diff
@@ -47,7 +47,10 @@ def make_random_trie(random):
 
 
 @given(random=strategies.randoms())
-@settings(max_examples=10)
+@settings(
+    max_examples=10,
+    deadline=4000,
```

Comment:
Interesting, was this taking a really long time with just a few examples?

Reply:
I think it's because it runs 10 examples, each which populate a trie with 1000 nodes, which as we know is a computationally expensive operation.